util/unifyfs/src/unifyfs.c

```diff
@@ -130,31 +131,33 @@ static void parse_cmd_arguments(int argc, char** argv)
     char* stage_in = NULL;
     char* stage_out = NULL;
 
+    int argument_count = 1;
+
     while ((ch = getopt_long(argc, argv,
-                             short_opts, long_opts, &optidx)) >= 0) {
-        switch (ch) {
-        case 'c':
-            printf("WARNING: cleanup not yet supported!\n");
-            cleanup = 1;
+			      short_opts, long_opts, &optidx)) >= 0) {
+	switch (ch) {
+	case 'c':
+	    printf("WARNING: cleanup not yet supported!\n");
+	    cleanup = 1;
             break;
 
-        case 'C':
-            consistency = unifyfs_cm_enum_from_str(optarg);
+	case 'C':
+	    consistency = unifyfs_cm_enum_from_str(optarg);
             if (consistency == UNIFYFS_CM_INVALID) {
-                usage(1);
+	        usage(1);
             }
             break;
 
-        case 'd':
+	case 'd':
             debug = 5;
             break;
 
-        case 'e':
-            srvr_exe = strdup(optarg);
+	case 'e':
+	    srvr_exe = strdup(optarg);
             break;
 
-        case 'm':
-            mountpoint = strdup(optarg);
+	case 'm':
+	    mountpoint = strdup(optarg);
```

Comment:
Fix indents.  It looks like you may be mixing tabs with spaces.

Reply:
This probably comes from me fixing the "code should start on a tabstop" in the patch checker.  Yes, emacs uses tabs/spaces relatively indiscriminately.  

I'm very sorry the diff looks ugly in the github display.  The source, however, looks fine as far as I can see and the extremely persnickety syntax checker is fine with it.  So unless there's a source file that's a problem when they check it out and use it in their local tools, I'm going to just leave this.

Reply:
@craigsteffen It is your responsibility to match the existing code style, which is 4-space indents (no tabs). You can easily set up your emacs config to do this (if you need help, just ask and I can provide the appropriate .emacsrc settings).

Reply:
I just want to clarify, I'm not suggesting you introduced all the tabs in this file. But, when you edit the lines, you should make sure the result only has 4-space indents. As a project, we really should be using a coding convention tool that automatically applies the desired style to all source files, so we can stop worrying about this in code reviews.

Reply:
I'll run astyle on this and repush it.

Reply:
I patched this in the copy that will be pushed soon.